spec/javascript/containers/LocationsContainerSpec.js

```diff
@@ -12,7 +12,7 @@ describe("LocationsContainer", () => {
     rating: 0,
     city: "Boston",
     state: "MA",
-    zip: "02111"
+    zip: "02111",
```

Comment:
Please undo this change -- there shouldn't be a comma after the last attribute in an object